.github/workflows/continuous-integration-workflow.yml

```diff
@@ -18,8 +19,16 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-20.04, windows-2019]
-        python-version: ['3.9', '3.10']
+        python-version: ['3.9', '3.10', '3.11', '3.12-dev']
         nox-session: [test, example]
+
+        exclude:
+          # Azure installation (Win, 3.12) currently hangs due to excessive pip backtracking
+          # Disable unit tests and retry in 2Q23
+          - python-version: '3.12-dev'
```

Comment:
Yes, it's still an issue, but only on Windows. I did some digging and it seems there are two dependencies that have issues installing.
azure-identify requires portalocker which requires pywin32. pywin32 added support for 3.12 in November and are testing against it, but haven't done a release since then, so I filed a bug with them.
Pillow added 3.12 support to their repo in February and plan to role it out with Pillow 10.0 by July. At that point there should be a pre-built binary in PyPI that will work for us.
There may be other issues beyond these, but these are currently what is preventing the installation of azure dependencies on Windows.

Reply:
Thanks for confirming. Can we update document to say explicitly that LISA doesn't support 3.12. It doesn't need to include the details. Just avoid users to try LISA with Python 3.12, and waste their time.

Reply:
Do we need that? Until it's GA then it's normal to assume it's not fully supported and if a downstream user finds an issue with 3.12, then we still want them to report it so we can track it.

Reply:
@avylove `LISA with Python 3.12 on windows` doesn't work, but `LISA with Python 3.12 on Linux` works?

Reply:
They both work now as far as the tests go. #2740